tests/aslts/bin/asltsrun

```diff
@@ -171,7 +171,7 @@ run_test_case()
 		AML_DONT_EXIST=$[ $AML_DONT_EXIST + 1 ]
 		TEST_RET=1
 	else
-		options="-ep"
+		options="-ep -el"
 		if [ $SLMODE == "norm" ]; then
```

Comment:
Weird.
I remember I have fixed this during the EFI task.
There really seem to be a big issue in the process.